go.mod

```diff
@@ -8,34 +8,36 @@ require (
 	github.com/campoy/embedmd v1.0.0
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
-	github.com/go-kit/kit v0.9.0
-	github.com/go-openapi/swag v0.19.5
+	github.com/go-kit/kit v0.10.0
+	github.com/go-openapi/swag v0.19.9
 	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.2
-	github.com/hashicorp/go-version v1.1.0
+	github.com/golang/protobuf v1.4.0
+	github.com/hashicorp/go-version v1.2.0
 	github.com/jsonnet-bundler/jsonnet-bundler v0.2.0
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
 	github.com/mitchellh/hashstructure v0.0.0-20170609045927-2bca23e0e452
-	github.com/oklog/run v1.0.0
+	github.com/oklog/run v1.1.0
 	github.com/openshift/prom-label-proxy v0.1.1-0.20191016113035-b8153a7f39f1
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.2.1
-	github.com/prometheus/prometheus v2.3.2+incompatible
-	github.com/stretchr/testify v1.4.0
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.5.1
+	github.com/prometheus/prometheus v1.8.2-0.20200507164740-ecee9c8abfd1
```

Comment:
I wish there was a better way to do this, it seems weird to show the version as 1.8.2 but point to 2.18.1.  I guess we can fix this in a separate PR if there is any better way to do it.

Reply:
AFAIK there's nothing we can do here for now because `github.com/prometheus/prometheus` doesn't respect the Go modules contract (e.g. the module name should be `github.com/prometheus/prometheus/v2`). There's an upstream effort to split the Prometheus code base into smaller modules which would be tagged as v0.x to avoid  (similar to `k8s.io/client-go` for instance) but there's nothing available yet.